packages/components/checkbox/src/Checkbox.tsx

```diff
@@ -1,5 +1,5 @@
 import * as CheckboxPrimitive from '@radix-ui/react-checkbox'
-import { Check } from '@spark-ui/icons'
+import { Check } from '@spark-ui/icons/src/icons/Check'
```

Comment:
We must not use relative imports from packages, it would only work locally. We must find a way to import from what is generated in the `dist` folder of `icons` (`src/icons` does not exists)